lib/utils/report.js

```diff
@@ -4,9 +4,11 @@ const _ = require('lodash');
 
 /** @typedef {{
   ruleName: string,
-  result: Object,
+  result: import('stylelint').PostcssResult,
   message: string,
-  node: Object,
+  node: import('postcss').Node & {
+    positionBy(opts: { index?: number, word?: string }): { line: number, column: number }
+  },
```

Comment:
PostCSS type definitions do not publish `positionBy()`.

https://github.com/postcss/postcss/blob/7.0.26/lib/node.es6#L411